lib/src/main/java/com/tubitv/media/utilities/EventLogger.java

```diff
@@ -214,6 +215,8 @@ public void onTimelineChanged(EventTime eventTime, final int reason) {
     @Override
     public void onPlayerError(EventTime eventTime, ExoPlaybackException e) {
         Log.e(TAG, "playerFailed [" + getSessionTimeString() + "]", e);
+        // If this is DRM, we'll just retry next one
+        PlayerContainer.repreparePlayerForNextVideoResource();
```

Comment:
I would suggest to keep this in a separate Class, create a new instance of this for drm and playback fail logic management